CODING_STANDARDS.MD

```diff
@@ -0,0 +1,24 @@
+# Coding Standards 
+
+### General Best Practices
+- Make sure to add unit test cases when adding new functionality and ensure added code is written in a testable manner.
+- All functions should have a description of their purpose and details of the parameters.
+- Soft limit of 100 characters per line.
+- Each issue should have one pull request unless the changes are only 1-3 lines.
```

Comment:
I meant a separate pull request is not needed if the changes are only 1-3 lines and can be part of another issue. It may be acceptable to include the code changes in an existing relevant pull request instead of creating a new one. Please let me know if it has to be updated.